packages/client/src/helpers/client/interceptors/authentication/types/TokenManagerOptions.types.ts

```diff
@@ -0,0 +1,59 @@
+import type { AxiosRequestConfig } from 'axios';
+import type { ITokenData } from '../token-providers/types/TokenData.types';
+
+export interface OptionsStorageProvider {
+  getItem(key: string): Promise<string | null> | string | null;
+  setItem(key: string, value: string): Promise<void> | void;
+  removeItem(key: string): Promise<void> | void;
+}
+
+export interface OptionsStorageSerializer {
+  serializeTokenData(tokenData: ITokenData | null): string;
+  deserializeTokenData(rawData: string, secretKey?: string): ITokenData;
+}
+
+export interface Options {
```

Comment:
This can be renamed as `AxiosAuthencationTokenManagerOptions` to be more specific.

Reply:
done.